host-provision.yml

```diff
@@ -58,6 +58,12 @@
 
 - name: Perform config tasks (SSH equivalence validation, LVM setup, etc)
   hosts: all
+  vars_prompt:
```

Comment:
Can we make sure this prompt only happens when the host is running RHEL?  (And not, say, Oracle LInux?)

Reply:
It may even be worth checking for an existing registration, in which case the prompt is unnecessary.

Reply:
This required a slight code refactoring.
Both of these points have been coded and tested.

The refactored code flow is as in this schematic: https://screenshot.googleplex.com/7uWan6c5N5Mrvyx  (internal)